contractsSol5/mock/MaliciousMatchingEngine.sol

```diff
@@ -8,13 +8,13 @@ contract MaliciousMatchingEngine is KyberMatchingEngine {
         KyberMatchingEngine(_admin)
     { /* empty body */ }
 
-    function getIsFeeAccountedReserves(bytes8[] memory reserveIds) internal view
-        returns(bool[] memory feeAccountedArr)
-    {
-        feeAccountedArr = new bool[](reserveIds.length);
+    // function getIsFeeAccountedReserves(bytes8[] memory reserveIds) internal view
```

Comment:
why not delete?

Reply:
we might need to read those when it come to coverage test